### PR TITLE
Add logging for demo apps

### PR DESCRIPTION
**Pull request by dineshg13**

### CODE REVIEW

File: charts/opentelemetry-demo-datadog/otelcol-config.yaml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        parse_to: body
+      - id: parse_log
+        type: json_parser
+        parse_from: body.log
+      - id: trace
+        type: trace_parser
+        trace_id:
+          parse_from: attributes.contextMap.trace_id
+        span_id:
+          parse_from: attributes.contextMap.span_id
```
Comment (gbbr) [inline charts/opentelemetry-demo-datadog/otelcol-config.yaml:61]: Next step should be getting rid of this. We need to make this as easy for the user as possible. This type of configuration in the documentation does not look very well.

Comment (dineshg13) [reply charts/opentelemetry-demo-datadog/otelcol-config.yaml:61]: yup .. agree .. we can follow the pattern are using for log level & message ..

File: charts/opentelemetry-demo-datadog/otelcol-config.yaml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    config:
+      scrape_configs:
+        - job_name: "otelcol"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: ".*grpc_io.*"
+              action: drop
```
Comment (gbbr) [inline charts/opentelemetry-demo-datadog/otelcol-config.yaml:12]: We don't know (AFAIU) why we need this, so should we remove it?

File: charts/opentelemetry-demo-datadog/values.yaml [hunk 3]
```diff
@@ -2,9 +2,10 @@ deployMode: "deployment"
 useOperator: false
 
 image:
-  repository:  otel/demo
-  otelcolversion: 0.59.0
+  repository:  dineshgurumurthydd/oteldemo
```
Comment (gbbr) [inline charts/opentelemetry-demo-datadog/values.yaml:5]: We should have this on our otel org asap. Can you talk to #eee?

**Review by gbbr — APPROVED** [review]
LGTM

File: charts/opentelemetry-demo-datadog/values.yaml (hunk 3 above)
Comment (dineshg13) [reply charts/opentelemetry-demo-datadog/values.yaml:5]: Will do . It is part of our work for ci/cd we want to tackle this quarter .

Comment (gbbr) [reply charts/opentelemetry-demo-datadog/values.yaml:5]: Surely we don't need a whole quarter to add a Docker repository to our org.

Comment (dineshg13) [reply charts/opentelemetry-demo-datadog/values.yaml:5]: I meant this work is part of CI/CD we want for our repos. I don't think we should block this PR for that . It is not gonna in next 2 weeks .

Comment (gbbr) [reply charts/opentelemetry-demo-datadog/values.yaml:5]: Agreed. It's a non-blocker. I have approved the PR. It was just a comment.

**Pull request merged**